scripts/docs/sources2html.pl

```diff
@@ -495,30 +497,17 @@ sub source_table {
               $version = "-";
           }
 
-          # Set description
           $description = "Variants imported from EVA";
         }
 
-        # Assuming only one config will have use_as_source set per species
-        if ( grep /^genotype$/, @types){
-          # Update source name and url to study id if possible
-          if ($source_name =~ /^(?!PRJ)/){
-            # Try getting the study id from database if not in vcf collection
-            my $sth3 = get_connection_and_query($db_name, $hostname, $sql_display_group);
```

Comment:
The study ID is always in the VCF collection now?

Reply:
No, but it is there we want to have the study url instead of EVA url (in which case the source name will be `"EVA"`)

Reply:
Okay, thanks!